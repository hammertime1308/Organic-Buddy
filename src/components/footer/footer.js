import React from 'react';
import { Image } from 'react-native';

import {
  TextSelected,
  TextUnselected,
  StyledFooter,
  NormalView,
  SelectedView,
} from './styles';

export const Footer = ({ selected = '' }) => {
  return (
    <StyledFooter>
      {selected === 'dashboard' ? (
        <SelectedView activeOpacity={0.8} onPress={() => {}}>
          <Image
            style={{ width: 35, height: 35, marginTop: 3, marginBottom: 5 }}
            source={require('../../assets/icons/dashboard.png')}
          />
          <TextSelected>Dashboard</TextSelected>
        </SelectedView>
      ) : (
        <NormalView
          activeOpacity={0.8}
          onPress={() => alert('dashboard pressed')}>
          <Image
            style={{ width: 35, height: 35, marginTop: 3, marginBottom: 5 }}
            source={require('../../assets/icons/dashboard.png')}
          />
          <TextUnselected>Dashboard</TextUnselected>
        </NormalView>
      )}

      {selected === 'shop' ? (
        <SelectedView activeOpacity={0.8} onPress={() => {}}>
          <Image
            style={{ width: 35, height: 35, marginTop: 3, marginBottom: 5 }}
            source={require('../../assets/icons/shop.png')}
          />
          <TextSelected>Shop</TextSelected>
        </SelectedView>
      ) : (
        <NormalView activeOpacity={0.8} onPress={() => alert('Shop pressed')}>
          <Image
            style={{ width: 35, height: 35, marginTop: 3, marginBottom: 5 }}
            source={require('../../assets/icons/shop.png')}
          />
          <TextUnselected>Shop</TextUnselected>
        </NormalView>
      )}

      {selected === 'getHelp' ? (
        <SelectedView activeOpacity={0.8} onPress={() => {}}>
          <Image
            style={{ width: 35, height: 35, marginTop: 3, marginBottom: 5 }}
            source={require('../../assets/icons/getHelp.png')}
          />
          <TextSelected>Get Help</TextSelected>
        </SelectedView>
      ) : (
        <NormalView
          activeOpacity={0.8}
          onPress={() => alert('Get Help pressed')}>
          <Image
            style={{ width: 35, height: 35, marginTop: 3, marginBottom: 5 }}
            source={require('../../assets/icons/getHelp.png')}
          />
          <TextUnselected>Get Help</TextUnselected>
        </NormalView>
      )}

      {selected === 'learn' ? (
        <SelectedView activeOpacity={0.8} onPress={() => {}}>
          <Image
            style={{ width: 35, height: 35, marginTop: 3, marginBottom: 5 }}
            source={require('../../assets/icons/learn.png')}
          />
          <TextSelected>Learn</TextSelected>
        </SelectedView>
      ) : (
        <NormalView activeOpacity={0.8} onPress={() => alert('Learn pressed')}>
          <Image
            style={{ width: 35, height: 35, marginTop: 3, marginBottom: 5 }}
            source={require('../../assets/icons/learn.png')}
          />
          <TextUnselected>Learn</TextUnselected>
        </NormalView>
      )}
    </StyledFooter>
  );
};
