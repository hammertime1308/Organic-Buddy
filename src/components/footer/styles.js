import styled from 'styled-components/native';

import { COLOR, FONT } from '../../styles';

export const StyledFooter = styled.View`
  height: 10%;
  display: flex;
  flex-direction: row;
  justify-content: space-evenly;
  background-color: ${COLOR.FooterUnselectedBackground};
`;

export const NormalView = styled.TouchableOpacity`
  background-color: ${COLOR.FooterUnselectedBackground};
  flex: 1;
  justify-content: center;
  align-items: center;
  margin: 5px;
`;

export const SelectedView = styled.TouchableOpacity`
  background-color: ${COLOR.GreenHeader};
  flex: 1;
  justify-content: center;
  align-items: center;
  margin: 5px;
  border-radius: 6px;

  elevation: 3;
`;

export const TextUnselected = styled.Text`
  font-size: ${FONT.FontSizeSmall};
  font-weight: ${FONT.FontWeightBold};
  width: 100%;
  text-align: center;
  color: ${COLOR.GreenHeader};
`;

export const TextSelected = styled.Text`
  font-size: ${FONT.FontSizeSmall};
  font-weight: ${FONT.FontWeightBold};
  color: ${COLOR.WhiteSelectedText};
  width: 100%;
  text-align: center;
`;
