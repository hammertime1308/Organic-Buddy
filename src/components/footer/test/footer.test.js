import React from 'react';
import renderer from 'react-test-renderer';
import { Footer } from '../footer';

describe('Footer', () => {
  it('Should render Footer without selection', () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('Should render Footer with dashboard selected', () => {
    const tree = renderer.create(<Footer selected="dashboard" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('Should render Footer with shop selected', () => {
    const tree = renderer.create(<Footer selected="shop" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('Should render Footer with get help selected', () => {
    const tree = renderer.create(<Footer selected="getHelp" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('Should render Footer with learn selected', () => {
    const tree = renderer.create(<Footer selected="learn" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
