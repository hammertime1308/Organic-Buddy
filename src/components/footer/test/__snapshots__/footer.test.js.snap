// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Should render Footer with dashboard selected 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "rgba(240, 240, 240,0.8)",
        "borderColor": "rgba(76, 154, 42, 1)",
        "borderTopWidth": 2,
        "display": "flex",
        "flexDirection": "row",
        "height": "10%",
        "justifyContent": "space-evenly",
      },
    ]
  }
>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "rgba(76, 154, 42, 1)",
        "borderRadius": 6,
        "elevation": 3,
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
        "marginBottom": 5,
        "marginLeft": 5,
        "marginRight": 5,
        "marginTop": 5,
        "opacity": 1,
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../src/assets/icons/dashboard.png",
        }
      }
      style={
        Object {
          "height": 35,
          "marginBottom": 5,
          "marginTop": 3,
          "width": 35,
        }
      }
    />
    <Text
      style={
        Array [
          Object {
            "color": "rgba(255, 255, 255, 1)",
            "fontSize": 12,
            "fontWeight": "700",
            "textAlign": "center",
            "width": "100%",
          },
        ]
      }
    >
      Dashboard
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "rgba(240, 240, 240,0.8)",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
        "marginBottom": 5,
        "marginLeft": 5,
        "marginRight": 5,
        "marginTop": 5,
        "opacity": 1,
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../src/assets/icons/shop.png",
        }
      }
      style={
        Object {
          "height": 35,
          "marginBottom": 5,
          "marginTop": 3,
          "width": 35,
        }
      }
    />
    <Text
      style={
        Array [
          Object {
            "color": "rgba(76, 154, 42, 1)",
            "fontSize": 12,
            "fontWeight": "700",
            "textAlign": "center",
            "width": "100%",
          },
        ]
      }
    >
      Shop
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "rgba(240, 240, 240,0.8)",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
        "marginBottom": 5,
        "marginLeft": 5,
        "marginRight": 5,
        "marginTop": 5,
        "opacity": 1,
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../src/assets/icons/getHelp.png",
        }
      }
      style={
        Object {
          "height": 35,
          "marginBottom": 5,
          "marginTop": 3,
          "width": 35,
        }
      }
    />
    <Text
      style={
        Array [
          Object {
            "color": "rgba(76, 154, 42, 1)",
            "fontSize": 12,
            "fontWeight": "700",
            "textAlign": "center",
            "width": "100%",
          },
        ]
      }
    >
      Get Help
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "rgba(240, 240, 240,0.8)",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
        "marginBottom": 5,
        "marginLeft": 5,
        "marginRight": 5,
        "marginTop": 5,
        "opacity": 1,
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../src/assets/icons/learn.png",
        }
      }
      style={
        Object {
          "height": 35,
          "marginBottom": 5,
          "marginTop": 3,
          "width": 35,
        }
      }
    />
    <Text
      style={
        Array [
          Object {
            "color": "rgba(76, 154, 42, 1)",
            "fontSize": 12,
            "fontWeight": "700",
            "textAlign": "center",
            "width": "100%",
          },
        ]
      }
    >
      Learn
    </Text>
  </View>
</View>
`;

exports[`Footer Should render Footer with get help selected 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "rgba(240, 240, 240,0.8)",
        "borderColor": "rgba(76, 154, 42, 1)",
        "borderTopWidth": 2,
        "display": "flex",
        "flexDirection": "row",
        "height": "10%",
        "justifyContent": "space-evenly",
      },
    ]
  }
>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "rgba(240, 240, 240,0.8)",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
        "marginBottom": 5,
        "marginLeft": 5,
        "marginRight": 5,
        "marginTop": 5,
        "opacity": 1,
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../src/assets/icons/dashboard.png",
        }
      }
      style={
        Object {
          "height": 35,
          "marginBottom": 5,
          "marginTop": 3,
          "width": 35,
        }
      }
    />
    <Text
      style={
        Array [
          Object {
            "color": "rgba(76, 154, 42, 1)",
            "fontSize": 12,
            "fontWeight": "700",
            "textAlign": "center",
            "width": "100%",
          },
        ]
      }
    >
      Dashboard
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "rgba(240, 240, 240,0.8)",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
        "marginBottom": 5,
        "marginLeft": 5,
        "marginRight": 5,
        "marginTop": 5,
        "opacity": 1,
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../src/assets/icons/shop.png",
        }
      }
      style={
        Object {
          "height": 35,
          "marginBottom": 5,
          "marginTop": 3,
          "width": 35,
        }
      }
    />
    <Text
      style={
        Array [
          Object {
            "color": "rgba(76, 154, 42, 1)",
            "fontSize": 12,
            "fontWeight": "700",
            "textAlign": "center",
            "width": "100%",
          },
        ]
      }
    >
      Shop
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "rgba(76, 154, 42, 1)",
        "borderRadius": 6,
        "elevation": 3,
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
        "marginBottom": 5,
        "marginLeft": 5,
        "marginRight": 5,
        "marginTop": 5,
        "opacity": 1,
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../src/assets/icons/getHelp.png",
        }
      }
      style={
        Object {
          "height": 35,
          "marginBottom": 5,
          "marginTop": 3,
          "width": 35,
        }
      }
    />
    <Text
      style={
        Array [
          Object {
            "color": "rgba(255, 255, 255, 1)",
            "fontSize": 12,
            "fontWeight": "700",
            "textAlign": "center",
            "width": "100%",
          },
        ]
      }
    >
      Get Help
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "rgba(240, 240, 240,0.8)",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
        "marginBottom": 5,
        "marginLeft": 5,
        "marginRight": 5,
        "marginTop": 5,
        "opacity": 1,
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../src/assets/icons/learn.png",
        }
      }
      style={
        Object {
          "height": 35,
          "marginBottom": 5,
          "marginTop": 3,
          "width": 35,
        }
      }
    />
    <Text
      style={
        Array [
          Object {
            "color": "rgba(76, 154, 42, 1)",
            "fontSize": 12,
            "fontWeight": "700",
            "textAlign": "center",
            "width": "100%",
          },
        ]
      }
    >
      Learn
    </Text>
  </View>
</View>
`;

exports[`Footer Should render Footer with learn selected 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "rgba(240, 240, 240,0.8)",
        "borderColor": "rgba(76, 154, 42, 1)",
        "borderTopWidth": 2,
        "display": "flex",
        "flexDirection": "row",
        "height": "10%",
        "justifyContent": "space-evenly",
      },
    ]
  }
>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "rgba(240, 240, 240,0.8)",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
        "marginBottom": 5,
        "marginLeft": 5,
        "marginRight": 5,
        "marginTop": 5,
        "opacity": 1,
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../src/assets/icons/dashboard.png",
        }
      }
      style={
        Object {
          "height": 35,
          "marginBottom": 5,
          "marginTop": 3,
          "width": 35,
        }
      }
    />
    <Text
      style={
        Array [
          Object {
            "color": "rgba(76, 154, 42, 1)",
            "fontSize": 12,
            "fontWeight": "700",
            "textAlign": "center",
            "width": "100%",
          },
        ]
      }
    >
      Dashboard
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "rgba(240, 240, 240,0.8)",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
        "marginBottom": 5,
        "marginLeft": 5,
        "marginRight": 5,
        "marginTop": 5,
        "opacity": 1,
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../src/assets/icons/shop.png",
        }
      }
      style={
        Object {
          "height": 35,
          "marginBottom": 5,
          "marginTop": 3,
          "width": 35,
        }
      }
    />
    <Text
      style={
        Array [
          Object {
            "color": "rgba(76, 154, 42, 1)",
            "fontSize": 12,
            "fontWeight": "700",
            "textAlign": "center",
            "width": "100%",
          },
        ]
      }
    >
      Shop
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "rgba(240, 240, 240,0.8)",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
        "marginBottom": 5,
        "marginLeft": 5,
        "marginRight": 5,
        "marginTop": 5,
        "opacity": 1,
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../src/assets/icons/getHelp.png",
        }
      }
      style={
        Object {
          "height": 35,
          "marginBottom": 5,
          "marginTop": 3,
          "width": 35,
        }
      }
    />
    <Text
      style={
        Array [
          Object {
            "color": "rgba(76, 154, 42, 1)",
            "fontSize": 12,
            "fontWeight": "700",
            "textAlign": "center",
            "width": "100%",
          },
        ]
      }
    >
      Get Help
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "rgba(76, 154, 42, 1)",
        "borderRadius": 6,
        "elevation": 3,
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
        "marginBottom": 5,
        "marginLeft": 5,
        "marginRight": 5,
        "marginTop": 5,
        "opacity": 1,
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../src/assets/icons/learn.png",
        }
      }
      style={
        Object {
          "height": 35,
          "marginBottom": 5,
          "marginTop": 3,
          "width": 35,
        }
      }
    />
    <Text
      style={
        Array [
          Object {
            "color": "rgba(255, 255, 255, 1)",
            "fontSize": 12,
            "fontWeight": "700",
            "textAlign": "center",
            "width": "100%",
          },
        ]
      }
    >
      Learn
    </Text>
  </View>
</View>
`;

exports[`Footer Should render Footer with shop selected 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "rgba(240, 240, 240,0.8)",
        "borderColor": "rgba(76, 154, 42, 1)",
        "borderTopWidth": 2,
        "display": "flex",
        "flexDirection": "row",
        "height": "10%",
        "justifyContent": "space-evenly",
      },
    ]
  }
>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "rgba(240, 240, 240,0.8)",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
        "marginBottom": 5,
        "marginLeft": 5,
        "marginRight": 5,
        "marginTop": 5,
        "opacity": 1,
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../src/assets/icons/dashboard.png",
        }
      }
      style={
        Object {
          "height": 35,
          "marginBottom": 5,
          "marginTop": 3,
          "width": 35,
        }
      }
    />
    <Text
      style={
        Array [
          Object {
            "color": "rgba(76, 154, 42, 1)",
            "fontSize": 12,
            "fontWeight": "700",
            "textAlign": "center",
            "width": "100%",
          },
        ]
      }
    >
      Dashboard
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "rgba(76, 154, 42, 1)",
        "borderRadius": 6,
        "elevation": 3,
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
        "marginBottom": 5,
        "marginLeft": 5,
        "marginRight": 5,
        "marginTop": 5,
        "opacity": 1,
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../src/assets/icons/shop.png",
        }
      }
      style={
        Object {
          "height": 35,
          "marginBottom": 5,
          "marginTop": 3,
          "width": 35,
        }
      }
    />
    <Text
      style={
        Array [
          Object {
            "color": "rgba(255, 255, 255, 1)",
            "fontSize": 12,
            "fontWeight": "700",
            "textAlign": "center",
            "width": "100%",
          },
        ]
      }
    >
      Shop
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "rgba(240, 240, 240,0.8)",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
        "marginBottom": 5,
        "marginLeft": 5,
        "marginRight": 5,
        "marginTop": 5,
        "opacity": 1,
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../src/assets/icons/getHelp.png",
        }
      }
      style={
        Object {
          "height": 35,
          "marginBottom": 5,
          "marginTop": 3,
          "width": 35,
        }
      }
    />
    <Text
      style={
        Array [
          Object {
            "color": "rgba(76, 154, 42, 1)",
            "fontSize": 12,
            "fontWeight": "700",
            "textAlign": "center",
            "width": "100%",
          },
        ]
      }
    >
      Get Help
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "rgba(240, 240, 240,0.8)",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
        "marginBottom": 5,
        "marginLeft": 5,
        "marginRight": 5,
        "marginTop": 5,
        "opacity": 1,
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../src/assets/icons/learn.png",
        }
      }
      style={
        Object {
          "height": 35,
          "marginBottom": 5,
          "marginTop": 3,
          "width": 35,
        }
      }
    />
    <Text
      style={
        Array [
          Object {
            "color": "rgba(76, 154, 42, 1)",
            "fontSize": 12,
            "fontWeight": "700",
            "textAlign": "center",
            "width": "100%",
          },
        ]
      }
    >
      Learn
    </Text>
  </View>
</View>
`;

exports[`Footer Should render Footer without selection 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "rgba(240, 240, 240,0.8)",
        "borderColor": "rgba(76, 154, 42, 1)",
        "borderTopWidth": 2,
        "display": "flex",
        "flexDirection": "row",
        "height": "10%",
        "justifyContent": "space-evenly",
      },
    ]
  }
>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "rgba(240, 240, 240,0.8)",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
        "marginBottom": 5,
        "marginLeft": 5,
        "marginRight": 5,
        "marginTop": 5,
        "opacity": 1,
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../src/assets/icons/dashboard.png",
        }
      }
      style={
        Object {
          "height": 35,
          "marginBottom": 5,
          "marginTop": 3,
          "width": 35,
        }
      }
    />
    <Text
      style={
        Array [
          Object {
            "color": "rgba(76, 154, 42, 1)",
            "fontSize": 12,
            "fontWeight": "700",
            "textAlign": "center",
            "width": "100%",
          },
        ]
      }
    >
      Dashboard
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "rgba(240, 240, 240,0.8)",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
        "marginBottom": 5,
        "marginLeft": 5,
        "marginRight": 5,
        "marginTop": 5,
        "opacity": 1,
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../src/assets/icons/shop.png",
        }
      }
      style={
        Object {
          "height": 35,
          "marginBottom": 5,
          "marginTop": 3,
          "width": 35,
        }
      }
    />
    <Text
      style={
        Array [
          Object {
            "color": "rgba(76, 154, 42, 1)",
            "fontSize": 12,
            "fontWeight": "700",
            "textAlign": "center",
            "width": "100%",
          },
        ]
      }
    >
      Shop
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "rgba(240, 240, 240,0.8)",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
        "marginBottom": 5,
        "marginLeft": 5,
        "marginRight": 5,
        "marginTop": 5,
        "opacity": 1,
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../src/assets/icons/getHelp.png",
        }
      }
      style={
        Object {
          "height": 35,
          "marginBottom": 5,
          "marginTop": 3,
          "width": 35,
        }
      }
    />
    <Text
      style={
        Array [
          Object {
            "color": "rgba(76, 154, 42, 1)",
            "fontSize": 12,
            "fontWeight": "700",
            "textAlign": "center",
            "width": "100%",
          },
        ]
      }
    >
      Get Help
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "rgba(240, 240, 240,0.8)",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
        "marginBottom": 5,
        "marginLeft": 5,
        "marginRight": 5,
        "marginTop": 5,
        "opacity": 1,
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../src/assets/icons/learn.png",
        }
      }
      style={
        Object {
          "height": 35,
          "marginBottom": 5,
          "marginTop": 3,
          "width": 35,
        }
      }
    />
    <Text
      style={
        Array [
          Object {
            "color": "rgba(76, 154, 42, 1)",
            "fontSize": 12,
            "fontWeight": "700",
            "textAlign": "center",
            "width": "100%",
          },
        ]
      }
    >
      Learn
    </Text>
  </View>
</View>
`;
